packages/protocol/src/action/request/Invoke.ts

```diff
@@ -241,9 +295,9 @@ export namespace Invoke {
             if (command === undefined) {
                 throw new MalformedRequestError(`Cluster ${cluster.name} does not define command ${request.command}`);
             }
-            return command as Specifier.CommandFor<Specifier.ClusterOf<R>, R["command"]>;
+            return command;
         }
-        return request.command as Specifier.CommandFor<Specifier.ClusterOf<R>, R["command"]>;
+        return request.command;
```

Comment:
It looks like this may have been intended to provider better typing for the result, is it possible to remove the `ClusterType.Command` return type on the function instead?